problems/http.go

```diff
@@ -31,10 +31,13 @@ func WriteResponse(ctx context.Context, err error, w http.ResponseWriter, r *htt
 		WithError(err).
 		WithField("code", statusCode)
 
-	// Log as an Error if statusCode is 5XX, otherwise as Info.
-	if statusCode/100 == http.StatusInternalServerError/100 {
+	// Log as an Error if statusCode is 5XX, Warn if context was cancelled, otherwise Info.
+	switch {
+	case statusCode/100 == http.StatusInternalServerError/100:
 		l.Error(problem.ProblemTitle())
-	} else {
+	case ctx.Err() == context.Canceled:
```

Comment:
You might need to do an errors.Is here?

Reply:
Is it better to check ctx.Err rather than err?

Reply:
`ctx.Err()` will be `context.Canceled` (no wrapping) if it has been canceled; `err` might have been replaced by something else along the way.

Reply:
This assumes that it was the same context that you used for your request that got canceled. One example might be if someone spawns a context to handle a custom timeout.